r18/apply_r17_patch.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+
+set -ex
+
+for i in erlc shell
+do
+        diff -u ../r17/erl_parse.yrl ../r17/erl_parse_$i.yrl | patch -p2 -o erl_parse_$i.yrl
```

Comment:
I suppose it's to generate appropriate erl_parse.yrl files but why r17 patches are in r18 directory? Just a bit 😕 .